web-local/src/routes/(application)/connector/clickhouse/[name]/[database]/[table]/+page.svelte

```diff
@@ -23,8 +24,4 @@
   <title>Rill Developer | {table}</title>
 </svelte:head>
 
-<TablePreviewWorkspace
```

Comment:
Any reason to keep individual pages for different connectors?

Reply:
I originally did it because each database has its own specific hierarchy (see below), but actually we might be able to use a `[...catchall]` route. I'll try that after we get the release out.

![image](https://github.com/rilldata/rill/assets/14206386/e35072ef-cd0c-429b-b779-7c094b71f73f)